service/worker/workerdeployment/activities.go

```diff
@@ -81,10 +82,11 @@ func (a *Activities) SyncUnversionedRamp(
 	}
 	var taskQueueSyncs []*deploymentspb.SyncDeploymentVersionUserDataRequest_SyncUserData
 	for _, tqInfo := range currVersionInfo.GetTaskQueueInfos() {
+		// TODO: group by TQs by name and make only one sync req per name
```

Comment:
```suggestion
		// TODO (Carly): group by TQs by name and make only one sync req per name
```
unless you want to do it in this PR